consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/request/ClaimStatusUpdateRequest.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.moj.sdt.cmc.consumers.request;
+
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@ToString
+@Setter
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonPropertyOrder({
+    "caseManRef",
+    "respondentId",
+    "updateType",
+    "paidInFullDate",
+    "section38Compliancy"
+
+})
+public class ClaimStatusUpdateRequest {
```

Comment:
It might be worth creating a base class for the request classes even though it may have nothing on it.  This would provide the ability for them to be handled generically if the need arises.

Reply:
will be tricky with property ordering etc, better we keep it like this